common/bs_scrapers/extract_info.py

```diff
@@ -0,0 +1,18 @@
+def extract_info(soup, configs):
+    for link in soup.findAll("a"):
+        if link.get("href") is None:
+            continue
+        if not link["href"].startswith(configs.web_path):
+            continue
+        print(link.get("href"))
+        url = str(link["href"])
+        name = url[url.rindex("/") :]
+
+        with open("url_name.txt", "a+") as output:
+            # This isn't really needed, but it's nice to have when debug is True
```

Comment:
Are you familiar with Python's standard [logging](https://docs.python.org/3/howto/logging.html) module? Seems like it can accomplish this, if you just want a way to surface debugging info

Reply:
Fairly certain that is an outdated comment, as there is no debugging in that chunk of code. It's been removed with 19c608c